weblate/addons/webhooks.py

```diff
@@ -104,7 +104,13 @@ def build_webhook_payload(self, change: Change) -> dict[str, int | str | list[st
             data["component"] = change.component.slug
         if change.translation:
             data["translation"] = change.translation.language.code
-
+        if change.path_object and (category := change.path_object.category):
```

Comment:
`path_object` is not a good choice here, as it fails with `'Translation' object has no attribute 'category'`.
```suggestion
        if change.component and (category := change.component.category):
```